### ISSUE TITLE
Progress update for big files

### ISSUE THREAD

**Issue by jorinvo** [skddc/rs-backup#4]
With big files or a slow connection the download can easily seems stuck. 
Would be nice have progress updates for each file.

**Comment by raucao** [comment]
Yes, could do this via https://www.npmjs.com/package/progress (which I'm using in the rs-messages-importer util e.g.).
